website-custom-views/src/content/development/navigation.mdx

```diff
@@ -0,0 +1,56 @@
+---
+title: Navigation
```

Comment:
I would suggest to name this `Routing`, as Navigation usually refers to the menu navigation.

Reply:
Updated here: 9db9e6be40ffadb8edeec8b0c32a8a2962896c78